packages/lab/src/tooltip/Tooltip.tsx

```diff
@@ -1,131 +1,155 @@
+  JSXElementConstructor,
+  ReactElement,
   ReactNode,
-  useCallback,
+  Ref,
+  RefObject,
 } from "react";
+import {
+  makePrefixer,
+  useForkRef,
+  StatusIndicator,
+  ValidationStatus,
+} from "@salt-ds/core";
+import { UseFloatingUIProps } from "../popper";
 import { Portal, PortalProps } from "../portal";
-import { makePrefixer, StatusIndicator } from "@salt-ds/core";
-import { useWindow } from "../window";
-
+import { useTooltip, UseTooltipProps } from "./useTooltip";
 import "./Tooltip.css";
 
-// Keep in order of preference. First items are used as default
-
-export type TooltipStatus = "error" | "info" | "success" | "warning";
-
 const withBaseName = makePrefixer("saltTooltip");
 
-// FIXME: Fix types
-export interface TooltipRenderProp {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  Icon: any; // typeof Icon;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  getIconProps: any; // StateAndPropGetterFunction<IconProps>;
-}
-
 export interface TooltipProps
-  extends Omit<HTMLAttributes<HTMLDivElement>, "title">,
-    Pick<PortalProps, "disablePortal" | "container"> {
-  arrowProps?: ComponentProps<"div">;
+  extends Omit<HTMLAttributes<HTMLDivElement>, "text">,
+    PortalProps,
+    Pick<UseFloatingUIProps, "open" | "onOpenChange" | "placement"> {
+  children?: ReactElement<any, string | JSXElementConstructor<any>>;
   /**
-   * Removes the tooltip arrow.
+   * Whether to hide the tooltip arrow tip. Defaults to `false`.
    */
   hideArrow?: boolean;
   /**
-   * Whether to hide a state icon within the tooltip
+   * Whether to hide the state icon within the tooltip. Defaults to `false`.
    */
   hideIcon?: boolean;
   /**
-   * A callback function to render the tooltip content
-   * @param {function} getIcon getter for the icon based on the state
-   * @param {function} getIconProps getter for the icon properties
+   * Content displayed inside the tooltip. Can be a string or a React component.
    */
-  render?: (props: TooltipRenderProp) => ReactNode;
+  content: string | ReactNode;
   /**
    * A string to determine the current status of the tooltip
    */
-  status?: TooltipStatus;
-  title?: string;
+  status?: ValidationStatus;
   /**
    * This prop is used to help implement the accessibility logic.
    * If you don't provide this prop. It falls back to a randomly generated id.
    */
   id?: string;
-  open?: boolean;
+  /**
+   * Delay in miliseconds before the tooltip is shown
+   */
+  enterDelay?: number;
+  /**
+   * Delay in miliseconds before the tooltip is hidden
+   */
+  leaveDelay?: number;
+  /**
+   * Option to not display the tooltip. Can be used in conditional situations like text truncation.
+   */
+  disabled?: boolean;
+  /**
+   * Option to remove the hover listener
+   */
+  disableHoverListener?: boolean;
+  /**
+   * Option to remove the focus listener
+   */
+  disableFocusListener?: boolean;
+  // arrowProps?: HTMLAttributes<HTMLDivElement>;
```

Comment:
Yes. From what I could see 'arrowProps' was added so that we could use TooltipProps on the useTooltip returns. No need for it anymore. Unless I'm missing something...